tools/cpplint.py

```diff
@@ -3089,6 +3118,14 @@ def main():
                                          codecs.getwriter('utf8'),
                                          'replace')
 
+
+  ch = logging.StreamHandler(sys.stdout)
+  logger.addHandler(ch)
+  logger.setLevel(logging.INFO)
+
+  if _cpplint_state.output_format == 'tap':
+    logger.info('TAP Version 13')
```

Comment:
I think all TAP is case insensitive, but yeah –  the spec does use lowercase. I copied above from eslint output. Will update.